src/blockchains/erc20/lib/constants.ts

```diff
@@ -32,7 +34,20 @@ function checkEnvVariables() {
 
 checkEnvVariables();
 
-module.exports = {
+export {
+    BLOCK_INTERVAL,
```

Comment:
I can see we export constants as object but also export it as anonymous objes. Is there a need to export it both ways?

Reply:
No need, fixed.